### PR TITLE
Update API to support galleries as a top-level content type, document `alternateLang` limitation

### PR DESCRIPTION
**Pull request by srilq**
#### This resolves Clubhouse issue: [#11094](https://app.clubhouse.io/condenastinternational/story/11094/update-the-api-docs-to-support-standalone-galleries-and-add-alternatelang-limitation)

- [x] If there are schema changes, checked that there is a card in Clubhouse for updating the Schema Validator to reflect the changes
- [x] Checked that the new markup renders properly by running it locally

Changes:
* Adds support for galleries as a top-level content type.
* Adds limitation on `alternateLang` to only allow 0 or 1 items.
* Updates PR template.

I think the additions needed to the gallery model in [copilot-core-models-cni](https://github.com/CondeNast-Copilot/copilot-core-models-cni) will be:
* `lang`
* `alternateLang`
* `contributorsAuthor`
* `photosTout`

### CODE REVIEW

**Comment by srilq** [comment]
Links to view rendered files on branch:
* [[View index.html.md]](https://github.com/conde-nast-international/flyway-api-docs/blob/gallery-update/source/index.html.md)
* [[View _types.md]](https://github.com/conde-nast-international/flyway-api-docs/blob/gallery-update/source/includes/_types.md)
* [[View _api.md]](https://github.com/conde-nast-international/flyway-api-docs/blob/gallery-update/source/includes/_api.md)

**Comment by srilq** [comment]
Just made a small change to clarify that `alternateLang` can be empty.

**Comment by srilq** [comment]
@garryyao As far as I know there is no requirement for migrating `contributorsPhotographer` so this isn't in the Flyway schema.

**Comment by garryyao** [comment]
That's fair, I'm asking only because seeing it in the [content type docs](https://github.com/conde-nast-international/flyway-api-docs/blob/gallery-update/source/includes/_types.md)

**Comment by srilq** [comment]
@garryyao Oh, that looks like a mistake actually. Thanks, will look into this!

File: source/includes/_api.md [hunk 1]
```diff
@@ -401,3 +401,207 @@ ID | The ID of the native article to retrieve
...
+  "contributorsAuthor": [
+    {
+      "type": "contributor",
+      "id": "c3c7",
+      "name": "Jana Ackermann",
+      "tags": [],
+      "uri": "die-redaktion/jana-ackermann"
+    }
+  ],
+  "items": [
```
Comment (almostagile) [inline source/includes/_api.md]: These items are not valid. Each item is a slide that can override the caption of the photo, which in our model is provided as the `photo` property of the slide.

File: source/includes/_types.md [hunk 2]
```diff
@@ -329,6 +321,153 @@ Field | Type | Required | Description | Restrictions
...
+  "ledeCaption": "",
+  "socialTitle": "Universal bringt Madonna-Biopic ins Kino",
+  "socialDescription": "Film-Tipp zum Vormerken: das Madonna-Biopic \"Blonde Ambition\"\r\n",
+  "contentSource": "digital_syndication",
+  "canonicalUrl": "http://www.vogue.de/people-kultur/people-news/madonna-biopic-blonde-ambition-170425",
+  "promoHed": "Universal bringt Madonna-Biopic ins Kino",
+  "promoDek": "",
+  "contributorsAuthor": ["..."],
+  "photosLede": ["..."],
+  "photosTout": ["..."],
```
Comment (almostagile) [inline source/includes/_types.md]: Do we want to define these separately? At the moment the rels `photosTout` and `items` are both populated with the photos on the slides. I believe this is also what the Copilot gallery editor does.

Comment (srilq) [reply source/includes/_types.md]: We should match the Copilot editor functionality. 👍

**Comment by srilq** [comment]
@almostagile Oooh I didn't know that was how it worked. Cool!

**Comment by srilq** [comment]
@amostagile I’ve addressed your comments now.

File: source/index.html.md [hunk 3]
```diff
@@ -96,10 +96,10 @@ All replies from the API should include a top level JSON property `brand` which
...
-under a unique ID and must include the `lang` property (an [ISO 639-1 2 letter code](https://en.wikipedia.org/wiki/ISO_639-1)) indicating in 
+Support is available to migrate articles, native articles and galleries for which there is both an original and translated language
+variant each of which should be published under a distinct URI. Each language variant should be migrated _separately_
+under a unique ID and must include the `lang` property (an [ISO 639-1 2 letter code](https://en.wikipedia.org/wiki/ISO_639-1)) indicating in
 which language it is written. It should also include the `alternateLang` relationship containing a stub of the
-article in the alternative language(s).
+article/gallery in the alternative language.
 
-The [all articles](#get-all-articles) endpoint should return separate links for each language variant.
+The [all articles](#get-all-articles) and [all galleries](#get-all-galleries) endpoints should return a separate link for the language variant.
```
Comment (almostagile) [inline source/index.html.md]: You have changed the meaning of this sentence, though I'm not quite sure what it now means.

Comment (srilq) [reply source/index.html.md]: Haha ok 😅  I think what I mean to say is that the 2 endpoints should both return a link for the original entity and also the language-variant entity separately (if there is one). Could you suggest a better wording? I think it just needs to be a little more verbose.

Comment (almostagile) [reply source/index.html.md]:
```
The [all articles](#get-all-articles) and [all galleries](#get-all-galleries) endpoints should include links to both language variants of translated articles. Just including a reference in `alternateLang` does not guarantee full migration of the translation.
```

Comment (srilq) [reply source/index.html.md]: Yeah that's much better.

File: source/includes/_types.md [hunk 4]
```diff
@@ -492,19 +501,60 @@ Field | Type | Required | Description | Restrictions
...
 `type` | string | Yes | Content model name | Must be `gallery`
 `id` | string | Yes | Source CMS ID | [See Resource IDs above](#resource-ids)
-`inlineHref` | string | No (default `/galleries/${id}`) | Href used for article body gallery embed | [See Inline Embeds](#inline-embeds)
-`hed` | string | Yes | Title of the gallery | [Copilot Limited markdown](#markdown-text-fields). Supports Bold, Italic and Strikethrough
+`inlineHref` | string | No (default `/galleries/${id}`) | Href used for article body gallery embed | Only used if the gallery is embedded in an article. [See Inline Embeds](#inline-embeds)
+`brand` | string | Yes | Short hand CNI brand code | -
+`lang` | [ISO 639-1 2 letter code](https://en.wikipedia.org/wiki/ISO_639-1) | Yes | The language in which the gallery text properties are written (e.g. photo caption) | See also [alternative languages](#alternative-languages)
+`hed` | string | Yes | The gallery headline | [Copilot Limited markdown](#markdown-text-fields). Supports Bold, Italic and Strikethrough
+`dek` | string | Yes | The gallery subhead | [Copilot Limited markdown](#markdown-text-fields). Supports Bold, Italic, Strikethrough, Link and Lists
```
[the remark this answers is not in the document]
Comment (almostagile) [reply source/includes/_types.md]: My opinion, leave it consistent with article for now (required).

**Pull request merged**